rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -1420,6 +1420,35 @@ void should_RequestReviewTask() {
     assertThat(repModel.getState()).isEqualTo(TaskState.READY_FOR_REVIEW);
   }
 
+  @Test
+  void should_ForceRequestReviewTask() {
```

Comment:
What do you think about should_ForceRequestReview_When_CurrentUserIsNotTheOwner?

Reply:
done